IronScheme/IronScheme.Closures/IronScheme.Closures.csproj

```diff
@@ -21,13 +22,24 @@
   </PropertyGroup>
   <ItemGroup>
     <ProjectReference Include="..\Microsoft.Scripting\IronScheme.Scripting.csproj" />
+    <PackageReference Include="runtime.win-x64.Microsoft.NETCore.ILAsm" Version="8.0.0" GeneratePathProperty="true" ExcludeAssets="all" />
+    <PackageReference Include="runtime.win-x64.Microsoft.NETCore.ILDAsm" Version="8.0.0" GeneratePathProperty="true" ExcludeAssets="all" />
   </ItemGroup>
   <ItemGroup>
     <Content Include="IronScheme.Glue.il">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
   </ItemGroup>
   <Target Name="PostBuild" AfterTargets="PostBuildEvent">
-    <Exec Command="&quot;$(TargetFrameworkSDKToolsDirectory)ildasm&quot; /out:&quot;$(TargetDir)$(TargetName).il&quot; /NOBAR /LINENUM &quot;$(TargetPath)&quot;&#xD;&#xA;&quot;$(SolutionDir)TailCallMaker\$(OutDir)TailCallMaker.exe&quot; &quot;$(TargetDir)$(TargetName).il&quot;&#xD;&#xA;&quot;$(TargetedFrameworkDir)\ilasm&quot; /dll /PDB /NOLOGO /QUIET /out:&quot;$(TargetDir)$(TargetName).dll&quot; /res:&quot;$(TargetDir)$(TargetName).res&quot; &quot;$(TargetDir)$(TargetName).il&quot; &quot;$(TargetDir)IronScheme.Glue.il&quot;" />
+    <PropertyGroup Condition="$(UseNetFxIlTools) == True">
+      <ILAsmPath>$(TargetFrameworkSDKToolsDirectory)ilasm</ILAsmPath>
+      <ILDAsmPath>$(TargetFrameworkSDKToolsDirectory)ildasm</ILDAsmPath>
+    </PropertyGroup>
+    <PropertyGroup Condition="$(UseNetFxIlTools) != True">
+      <ILAsmPath>$(Pkgruntime_win-x64_Microsoft_NETCore_ILAsm)\runtimes\win-x64\native\ilasm</ILAsmPath>
+      <ILDAsmPath>$(Pkgruntime_win-x64_Microsoft_NETCore_ILDAsm)\runtimes\win-x64\native\ildasm</ILDAsmPath>
+    </PropertyGroup>
+    <Exec Condition="$(UseNetFxIlTools) == True" Command="&quot;$(TargetFrameworkSDKToolsDirectory)ildasm&quot; /out:&quot;$(TargetDir)$(TargetName).il&quot; /NOBAR /LINENUM &quot;$(TargetPath)&quot;&#xD;&#xA;&quot;$(SolutionDir)TailCallMaker\$(OutDir)TailCallMaker.exe&quot; &quot;$(TargetDir)$(TargetName).il&quot;&#xD;&#xA;&quot;$(TargetedFrameworkDir)\ilasm&quot; /dll /PDB /NOLOGO /QUIET /out:&quot;$(TargetDir)$(TargetName).dll&quot; /res:&quot;$(TargetDir)$(TargetName).res&quot; &quot;$(TargetDir)$(TargetName).il&quot; &quot;$(TargetDir)IronScheme.Glue.il&quot;" />

```

Comment:
All that magic is due to fact that `/res` is also don't supported by newer ILDasm.